src/index.html

```diff
@@ -1,13 +1,389 @@
+              <span>Clients</span>
+            </p>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="grid">
+          <a
+            href="#"
+            class="logo"
+            >
+            <img
+              src="images/Bakerlab.svg"
+              alt="logo"
+              class="logo__img logo__img--2 grid__item--tablet-1-4 grid__item--desktop-1-4">
+          </a>
+
+          <ul class="footer__list grid__item--tablet-1-4 grid__item--desktop-1-4">
+            <li class="footer__item gray">
+              Menu
+            </li>
+            <li class="footer__item">
+              <a href="#products" class="footer__link">
+                Products
+              </a>
+            </li>
+            <li class="footer__item">
+              <a href="#about" class="footer__link">
+                About Us
+              </a>
+            </li>
+            <li class="footer__item">
+              <a href="#" class="footer__link">
+                Blog
+              </a>
+            </li>
+            <li class="footer__item">
+              <a href="#" class="footer__link">
+                Terms & Privacy
+              </a>
+            </li>
+          </ul>
+
+          <p class="footer__contact-us grid__item--tablet-5-8 grid__item--desktop-5-8">
+            Contact us
+          </p>
+
+          <ul class="footer__list grid__item--tablet-5-8 grid__item--desktop-5-6">
+            <li class="footer__item gray">
+              Call us
+            </li>
+            <li class="footer__item">
+              <a href="tel:(629) 555-0129" class="footer__link">
+                (629) 555-0129
+              </a>
+            </li>
+            <li class="footer__item">
+              <a href="tel:(603) 555-0123" class="footer__link">
+                (603) 555-0123
+              </a>
+            </li>
+            <li class="footer__item">
+              <a href="tel:(704) 555-0127" class="footer__link">
+                (704) 555-0127
+              </a>
+            </li>
+          </ul>
+
+          <ul class="footer__list grid__item--tablet-5-8 grid__item--desktop-7-8">
+            <li class="footer__item">
+              <a href="mailto:bakerlab@example.com" class="footer__link gray">
+                Email us
+              </a>
+            </li>
+            <li class="footer__item">
+              <a href="mailto:bakerlab@example.com" class="footer__link">
+                bakerlab@example.com
+              </a>
+            </li>
+          </ul>
+
+          <div class="footer__social grid__item--tablet-1-4" id="blog">
+            <a
+            href="https://www.facebook.com/BakeryLab2014/"
+            target="_blank"
+            >
+              <img
+                src="images/Icon-facebook.svg"
+                alt="icon"
+                class="footer__icon">
+            </a>
```

Comment:
Reed about code styles by this [link](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)  - with great examples.
```suggestion
            <a
              href="https://www.facebook.com/BakeryLab2014/"
              target="_blank"
            >
              <img
                src="images/Icon-facebook.svg"
                alt="icon"
                class="footer__icon"
              >
            </a>
```